src/Paket.Core/PackageMetaData.fs

```diff
@@ -93,10 +115,10 @@ let merge templateFile metaData =
     match templateFile with
     | { Contents = ProjectInfo(md, opt) } -> 
         let merged = 
-            { Id = md.Id ++ metaData.Id
-              Version = md.Version ++ metaData.Version
-              Authors = md.Authors ++ metaData.Authors
-              Description = md.Description ++ metaData.Description }
+            { Id = metaData.Id ++ md.Id
```

Comment:
Yes, as I wasn't sure where else it might be used (and what the desired
effect would be)

On 23 February 2015 at 15:52, Steffen Forkmann <notifications@github.com>
wrote:

> In src/Paket.Core/PackageMetaData.fs
> <https://github.com/fsprojects/Paket/pull/644#discussion_r25174262>:
>
> > @@ -93,10 +115,10 @@ let merge templateFile metaData =
> >      match templateFile with
> >      | { Contents = ProjectInfo(md, opt) } ->
> >          let merged =
> > -            { Id = md.Id ++ metaData.Id
> > -              Version = md.Version ++ metaData.Version
> > -              Authors = md.Authors ++ metaData.Authors
> > -              Description = md.Description ++ metaData.Description }
> > +            { Id = metaData.Id ++ md.Id
>
> so you changed the usage and not the operator.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fsprojects/Paket/pull/644/files#r25174262>.
>